dev/build-docs.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# We use python2.7 for building the docs because Pygments.rb/Pygments is incompatible with
+# python 3, more info: https://github.com/jekyll/jekyll/issues/2604
+docker build -t databricks/graphframes --build-arg PYTHON_VERSION=2.7 .
+
+# build the docs image
+docker build -t databricks/graphframes-docs docs/
+
+# create the assembly jar on host because we have ivy/maven cache
+build/sbt 'set test in assembly := {}' assembly
```

Comment:
The `jekyll build` command will trigger `build/sbt clean compile doc` in `_plugins/copy_api_dirs.rb`. The assembly built here will get cleaned. Let's do the following:

* Remove this line.
* Mount "\~/.ivy2" and "\~/.sbt" to accelerate the build inside docker:

~~~
docker run --rm -v "$(pwd):/mnt/graphframes" -v "$HOME/.sbt:/root/.sbt" -v "$HOME/.ivy2:/root/.ivy2" ...
~~~